### PR TITLE
Add incident report: 2023-02-01 leap hub, dask-gateway induced outage

### PR DESCRIPTION
**Pull request by consideRatio**
Marked as draft pending time available to refine this further based on feedback!

### Related

- https://github.com/2i2c-org/infrastructure/issues/2126

### CODE REVIEW

File: reports/2023-02-01-daskhub-induced-evictions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# 2023-02-01 Heavy use of dask-gateway induced critical pod evictions
+
```
Comment (pnasrat) [inline reports/2023-02-01-daskhub-induced-evictions.md:2]: Please add summary statement and impact.

File: reports/2023-02-01-daskhub-induced-evictions.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Timeline
+
+All times in UTC+1
+
+- 2023-02-01 - [Summary of issue updated between ~8-9 PM](https://github.com/2i2c-org/infrastructure/issues/2126#issuecomment-1412554908)
```
Comment (pnasrat) [inline reports/2023-02-01-daskhub-induced-evictions.md:7]: Generally I'd expect to see a full timeline here not a link - eg https://sre.google/sre-book/example-postmortem/ 

However looking at the other incident reports in this repo it doesn't seem like we have a clear shared expectation of what should be here.

File: reports/2023-02-01-daskhub-induced-evictions.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## What went wrong
+
+- I believe various critical pods on core nodes pods got evicted when prometheus started scraping from ~200 nodes metrics exporters
```
Comment (pnasrat) [inline reports/2023-02-01-daskhub-induced-evictions.md:11]: Can we do better than belief? Is there not evidence to support your hypothesis here? That's what I'd expect the incident report/postmortem to reveal through analysis of the contributing factors/root causes/triggers.

File: reports/2023-02-01-daskhub-induced-evictions.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- I think its likely, but I can't say for sure, that the dask scheduler pod also would run into resource limitations with this amount of workers
+
+## Follow-up improvements
+
+- #2127
```
Comment (pnasrat) [inline reports/2023-02-01-daskhub-induced-evictions.md:16]: Please make these full markdown links with a summary line

File: reports/2023-02-01-daskhub-induced-evictions.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+## Timeline
+
+All times in UTC+1
+
+- 2023-02-01 - [Summary of issue updated between ~8-9 PM](https://github.com/2i2c-org/infrastructure/issues/2126#issuecomment-1412554908)
+
+## What went wrong
+
+- I believe various critical pods on core nodes pods got evicted when prometheus started scraping from ~200 nodes metrics exporters
+- I think its likely, but I can't say for sure, that the dask scheduler pod also would run into resource limitations with this amount of workers
```
Comment (pnasrat) [inline reports/2023-02-01-daskhub-induced-evictions.md:12]: Is this what went wrong? I'm not sure what this speculation is for in terms of analyzing the incident. Is this a lesson learned?

**Review by pnasrat — CHANGES_REQUESTED** [review]
I've left some comments. In general an incident report should contain more than just links back to the issue.

**Comment by consideRatio** [comment]
@pnasrat thank you for the review! These are all relevant points, I'll leave them unanswered for now since I'm feeling a time pressure to focus on other things.

**Pull request merged**